packages/components/src/components/card/card.css

```diff
@@ -16,7 +16,7 @@
   --radius: var(--telekom-radius-large);
   --box-shadow: var(--telekom-shadow-raised-standard);
   --box-shadow-hover: var(--telekom-shadow-raised-hover);
-  --box-shadow-focus: 0 0 0 var(--telekom-spacing-unit-x05)
+  --box-shadow-focus: 0 0 0 var(--telekom-line-weight-highlight)
```

Comment:
could this be a new token as well? seems like it's defined in the same way for a lot of components

Reply:
yep, I used this `line-weight…` in all borders and "focus shadows"